pyproject.toml

```diff
@@ -20,11 +21,11 @@ exclude = ["TestSuite/**", "*/tests/*", "tests"]
 demisto-sdk = "demisto_sdk.__main__:main"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = "^3.8,<3.12"
```

Comment:
@ilaner why changed from `>=` to `^`

Reply:
Both do the same, using the `^` notion is the standard way